recipes/stc/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.0.0-rc1":
```

Comment:
hmmm, this is the only tag...

Reply:
yes, I don't think it's a problem, there is just one release so far, more will appear in the future